src/test/java/be/seeseemelk/mockbukkit/scheduler/BukkitSchedulerMockTest.java

```diff
@@ -179,4 +180,91 @@ void runTaskTimerAsynchronously_TaskExecutedOnSeperateThread() throws Interrupte
 		assertFalse(scheduler.isQueued(testTask.getTaskId()));
 		assertEquals(2, count.get());
 	}
+
+	@Test
+	public void getPendingTasks_Sync()
+	{
+		assertEquals(0, scheduler.getPendingTasks().size());
```

Comment:
This might be better from a semantical standpoint.

```suggestion
		assertTrue(scheduler.getPendingTasks().isEmpty());
```